warp-core-gatling/src/main/scala/com/workday/warp/junit/GatlingTestExtension.scala

```diff
@@ -0,0 +1,44 @@
+package com.workday.warp.junit
+
+import com.workday.warp.adapters.gatling.WarpSimulation
+import io.gatling.app.Gatling
+import io.gatling.core.Predef._
+import io.gatling.core.config.{GatlingConfiguration, GatlingPropertiesBuilder}
+import io.gatling.core.structure.PopulationBuilder
+import org.pmw.tinylog.Logger
+
+import scala.util.{Failure, Success, Try}
+
+/**
+  * Created by tomas.mccandless on 7/1/20.
+  */
+trait GatlingTestExtensionLike extends WarpSimulation {
+
+
+  @UnitTest
+  def run(): Unit = {
+    Logger.info("running gatling")
+    Try {
+      val properties: GatlingPropertiesBuilder = new GatlingPropertiesBuilder
+      properties.simulationClass(this.getClass.getCanonicalName)
+//      properties.noReports()
+      properties.resultsDirectory("build/")
+      Gatling.fromMap(properties.build)
```

Comment:
cool you ended up getting the junit extension working?

Reply:
nope, sorry, i just forgot to delete that extension 😄  

i'll remove it